resources/profiles/MagicMaker/machine/MM hqs SF 0.4 nozzle.json

```diff
@@ -126,12 +115,24 @@
     ],
     "z_hop": [
         "0.4"
-    ],
-    "z_hop_types": [
-        "Spiral Lift"
     ],
 	"default_filament_profile": [
-		"Generic PLA @System"
+		"MM Generic PLA"

```

Comment:
This is wrong.
Please revert the change  as we don't have MM Generic PLA anymore, instead we have the global filament profile.
Please revert all similar changes in other places